docs/src/catalyst_applications/bifurcation_diagrams.md

```diff
@@ -1,115 +1,118 @@
+- `p_min` and `p_max`: Sets the interval over which the bifurcation diagram is computed (with the continuation stopping if it reaches these bounds).
+- `dsmin` and `dsmax`: The minimum and maximum length of the continuation steps (in the bifurcation parameter's value).
+- `max_steps`: The maximum number of continuation steps. If a bifurcation diagram looks incomplete, try increasing this value.
+- `newton_options`: Options for the Newton's method that BifurcationKit uses to find steady states. This can be created using `NewtonPar(tol = 1e-9, max_iterations = 100)` which here sets the tolerance (to `1e-9`) and the maximum number of newton iterations (to `100`).
 
-bprob = BifurcationProblem(F, oprob.u0, oprob.p, (@lens _[bif_idx]);
-                           record_from_solution = (x, p) -> x[plot_idx], J = J)
-nothing   # hide
-```
-Next, we need to specify the input options for the pseudo-arclength continuation
-method (PACM) which produces the diagram.
+The previous bifurcation diagram can be computed, with these various options specified, in the following way:
 ```@example ex1
-bopts = ContinuationPar(dsmax = 0.05,          # Max arclength in PACM.
-                        dsmin = 1e-4,          # Min arclength in PACM.
-                        ds = 0.001,            # Initial (positive) arclength in PACM.
-                        max_steps = 100000,     # Max number of steps.
-                        p_min = p_span[1],      # Min p-val (if hit, the method stops).
-                        p_max = p_span[2],      # Max p-val (if hit, the method stops).
-                        detect_bifurcation = 3) # Value in {0,1,2,3}
-nothing   # hide
+p_span = (2.0, 20.0)
+opt_newton = NewtonPar(tol = 1e-9, max_iterations = 1000)
+opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2], 
+                          dsmin = 0.001, dsmax = 0.01, max_steps = 1000,
+                          newton_options = opt_newton)
+bif_dia = bifurcationdiagram(bprob, PALC(), 2, (args...) -> opts_br; bothside=true)
+nothing # hide
 ```
-Here `detectBifurcation` determines to what extent bifurcation points are
-detected and how accurately their values are determined. Three indicates to use the most
-accurate method for calculating their values.
+(however, in this case these additional settings have no significant effect on the result)
 
-We are now ready to compute the bifurcation diagram:
+## Bifurcation diagrams with disjoint branches
+Let's consider the previous case, but instead compute the bifurcation diagram over the interval *(2.0, 15.0)*:
 ```@example ex1
-bf = bifurcationdiagram(bprob, PALC(), 2, (args...) -> bopts)
-nothing   # hide
+p_span = (2.0, 15.0)
+opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2], max_steps = 1000)
+bif_dia = bifurcationdiagram(bprob, PALC(), 2, (args...) -> opts_br; bothside=true)
```

Comment:
Maybe we should suggest to Romain a dispatch that if a `ContinuationPar` is passed directly as the fourth argument he just directly build the `(args...) -> opts_br` function? This would make things easier on users.

Reply:
agreed

Reply:
Hopefully this will happen soon-ish: https://github.com/bifurcationkit/BifurcationKit.jl/issues/132